nerves_defconfig

```diff
@@ -1,6 +1,7 @@
 BR2_arm=y
 BR2_cortex_a53=y
 BR2_ARM_FPU_NEON_VFPV4=y
+BR2_PRIMARY_SITE="https://s3.amazonaws.com/nerves-project/dl"
```

Comment:
I have mixed feelings about everyone hitting our S3 bucket first when they build. Let's chat about this. Maybe it's ok.